rocketpy/Function.py

```diff
@@ -2847,11 +3009,11 @@ def __get__(self, instance, owner=None):
                     # Handle methods which are the source themselves
                     source = lambda *_: self.func(instance, *_)
                     val = Function(source, *args, **kwargs)
-                except Exception:
-                    raise Exception(
-                        "Could not create Function object from method "
-                        f"{self.func.__name__}."
-                    )
+                # except Exception:
+                #     raise Exception(
+                #         "Could not create Function object from method "
+                #         f"{self.func.__name__}."
+                #     )
 
```

Comment:
Should this be commented?

Reply:
It shouldn't, it was a debugging workaround, I will fix that.

Reply:
Solved by [7bffaeb](https://github.com/RocketPy-Team/RocketPy/pull/354/commits/7bffaebd55c92f07cfd38a1c79de8e0e9eabbca6).